splitio/client/client.py

```diff
@@ -393,7 +407,13 @@ def track(self, key, traffic_type, event_type, value=None, properties=None):
             timestamp=utctime_ms(),
             properties=properties,
         )
-        return self._recorder.record_track_stats([EventWrapper(
+
+        return_flag = self._recorder.record_track_stats([EventWrapper(
             event=event,
             size=size,
         )])
+        self._telemetry_evaluation_producer.record_latency(TRACK, get_current_epoch_time() - start)
+        if not return_flag:
```

Comment:
this is not required, keep in mind that track can return false and that won't be an exception. we will only return exception only in captured exception calls

Reply:
Will add a try exception block.